client/src/components/Footnote.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { theme } from "../constants/theme";
+
+import styled from "styled-components";
+
+const { colors } = theme;
+
+const StyledFooter = styled.div`
+  position: absolute;
+  width: 37.6rem;
+  height: 7.1rem;
+  left: 0;
+  bottom: 0;
+  top: 75.9rem;
+
+  background: ${colors.selago};
+  text-align: center;
+  line-height: 1.5rem;
+  font-family: Poppins;
+  font-size: 1.1rem;
+`;
+
+const Copyright = styled.div`
+  position: absolute;
+  width: 31.2rem;
+  height: 2.1rem;
+  left: 2.8rem;
+  top: 1.8rem;
+
+  font-weight: 500;
+  color: ${colors.darkGray};
+`;
+
+const Policies = styled.div`
+  position: absolute;
```

Comment:
Don't think we need position absolute here -  it would naturally be under the `Copyright` component and by default have display block and take up the entire width